### ISSUE TITLE
Search dropdown on mobile no longer shows menu on first click in search input

### ISSUE THREAD

**Issue by jakenuts** [fomantic/Fomantic-UI#2176]
Update on June 24th (a75fbd2e540e164d2f18a6a93918ea33f65b0e57) introduced the 'focused' variable which is checked in the search:focus handler and it prevents showing the menu. The focus on mobile appears to happen in two stages, in the first check the document.activeElement is the body, then in the second stage the activeElement is the search input. Prior to the addition of this line in the search:focus handler the second attempt caused the dropdown to open. 

Working in 9/6/2020:

          search: {
            focus: function(event) {
              activated = true;
              if(module.is.multiple()) {
                module.remove.activeLabel();
              }
              if(settings.showOnFocus || (event.type !== 'focus' && event.type !== 'focusin')) {
                module.search();
              }
            },

Not working in 6/25/21:

          search: {
            focus: function(event) {
              activated = true;
              if(module.is.multiple()) {
                module.remove.activeLabel();
              }
              if(!focused && !module.is.active() && (settings.showOnFocus || (event.type !== 'focus' && event.type !== 'focusin'))) {
                focused = true;
                module.search();
              }
            },

Commenting out the "!focused && !module.is.active() &&" part of the condition restores the expected behavior (but I have no idea what the downside is).

Thanks!

James

https://github.com/fomantic/Fomantic-UI/blob/a75fbd2e540e164d2f18a6a93918ea33f65b0e57/dist/components/dropdown.js#L1134

**Comment by daveharris** [comment]
I've just found that I have the same issue. I've found that it also doesn't re-open the menu on subsequent clicks either. I've been able to reproduce it locally and also on the FomanticUI docs site. See attached screen recording (sorry for the poor quality, it has to be less than 10MB to upload).

https://user-images.githubusercontent.com/111275/153782560-52d8481c-0495-4b57-a166-5ef9c78c231f.mov

Weirdly while playing I was sometimes able to get the menu to display, but it was rather sporadic. Maybe it depends if you click on the control or the dropdown icon perhaps also?
